feat(serverless): advanced cold starts doc (#3891)
scaleway/docs-content: serverless/containers/concepts.mdx

## Cold Start

Cold start is the time a Container takes to handle a request when it is called for the first time.

Startup process steps are:
* Downloading the container image to our infrastructure

> * Starting the container. Optimise your container startup speed to minimize this step *(ex: avoid waiting for slow connections or downloading large objects etc... at statup)*

Reviewer: ```suggestion
* Starting the container. Optimize your container startup speed to minimize this step (e.g., avoid waiting for slow connections or downloading large objects at startup)
```

* Waiting for the containeer to listen on the configured port.

> [How to reduce cold start](/faq/serverless-containers/#how-can-i-reduce-the-cold-starts-of-serverless-containers)

Reviewer: ```suggestion
[How to reduce cold starst](/faq/serverless-containers/#how-can-i-reduce-the-cold-starts-of-serverless-containers)
```

> [How to reduce cold start](/faq/serverless-containers/#how-can-i-reduce-the-cold-starts-of-serverless-containers)

Reviewer 2: ```suggestion
[How to reduce cold starts](/faq/serverless-containers/#how-can-i-reduce-the-cold-starts-of-serverless-containers)
```
